src/app/containers/Article/index.jsx

```diff
@@ -48,43 +84,42 @@ class ArticleContainer extends Component {
       const data = await response.json();
       const amp = isAmpPath(path);
 
-      return { amp, data };
+      return { amp, data, service };
     } catch (error) {
       console.log(error); // eslint-disable-line no-console
       return {};
     }
   }
 
-  render() {
-    const { amp, data } = this.props;
-    /*
-      This handles async data fetching, and a 'loading state', which we should look to handle more intelligently.
+  /*
+    [1] This handles async data fetching, and a 'loading state', which we should look to handle more intelligently.
       After.JS gives no explicit loading state, so we just have to check for a lack of data.
       After.JS should handle async data fetching in a more informative way, which will be become an issue for error handling.
-    */
+  */
+  render() {
+    const { amp, data, service } = this.props;
     if (!data) {
-      return 'Loading...';
+      return 'Loading...'; /* [1] */
     }
-
     const { content, metadata, promo } = data;
-    const { id: aresArticleId } = metadata;
-
+    const { id: aresArticleId, tags } = metadata;
     const id = aresArticleId.split(':').pop();
-    const { blocks } = content.model;
-    const canonicalLink = `https://www.bbc.com/news/articles/${id}`;
-
-    const metadataProps = {
-      amp,
-      canonicalLink,
-      lang: metadata.passport.language,
-      title: promo.headlines.seoHeadline,
-    };
-
+    const config = serviceConfig[service];
     return (
       <Fragment>
         <Header />
-        <Metadata {...metadataProps} />
-        <MainContent blocks={blocks} />
+        <Metadata
+          {...metadataProps(
+            amp,
+            config,
+            id,
+            metadata,
+            metaTags(tags),
```

Comment:
It might be a bit cleaner to not extract `tags` and pass the concat'd tags in here, but concat `tags` inside `metadataProps`, could extract `tags` out of the `metadata` object that is passed in the line above.

Reply:
I've made this change.